aws/resource_aws_spot_instance_request_test.go

```diff
@@ -511,152 +511,152 @@ func TestAccAWSSpotInstanceRequest_InterruptHibernate(t *testing.T) {
+}
 
-	%v
+%v
 
-	resource "aws_spot_instance_request" "foo" {
-		ami = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
-		instance_type = "m1.small"
-		key_name = "${aws_key_pair.debugging.key_name}"
+resource "aws_spot_instance_request" "foo" {
+  ami           = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
+  instance_type = "m1.small"
+  key_name      = aws_key_pair.debugging.key_name
 
-		// base price is $0.044 hourly, so bidding above that should theoretically
-		// always fulfill
-		spot_price = "0.05"
+  // base price is $0.044 hourly, so bidding above that should theoretically
+  // always fulfill
+  spot_price = "0.05"
 
-		// we wait for fulfillment because we want to inspect the launched instance
-		// and verify termination behavior
-		wait_for_fulfillment = true
+  // we wait for fulfillment because we want to inspect the launched instance
+  // and verify termination behavior
+  wait_for_fulfillment = true
 
-	tags = {
-			Name = "terraform-test"
-		}
-	}
+  tags = {
+    Name = "terraform-test"
+  }
+}
 `, rInt, testAccLatestAmazonLinuxHvmEbsAmiConfig())
 }
 
 func testAccAWSSpotInstanceRequestConfigValidUntil(rInt int, validUntil string) string {
 	return fmt.Sprintf(`
-	resource "aws_key_pair" "debugging" {
-		key_name = "tmp-key-%d"
-		public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD3F6tyPEFEzV0LX3X8BsXdMsQz1x2cEikKDEY0aIj41qgxMCP/iteneqXSIFZBp5vizPvaoIR3Um9xK7PGoW8giupGn+EPuxIA4cDM4vzOqOkiMPhz5XK0whEjkVzTo4+S0puvDZuwIsdiW9mxhJc7tgBNL0cYlWSYVkz4G/fslNfRPW5mYAM49f4fhtxPb5ok4Q2Lg9dPKVHO/Bgeu5woMc7RY0p1ej6D4CKFE6lymSDJpW0YHX/wqE9+cfEauh7xZcG0q9t2ta6F6fmX0agvpFyZo8aFbXeUBr7osSCJNgvavWbM/06niWrOvYX2xwWdhXmXSrbX8ZbabVohBK41 phodgson@thoughtworks.com"
-	}
+resource "aws_key_pair" "debugging" {
+  key_name   = "tmp-key-%d"
+  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQD3F6tyPEFEzV0LX3X8BsXdMsQz1x2cEikKDEY0aIj41qgxMCP/iteneqXSIFZBp5vizPvaoIR3Um9xK7PGoW8giupGn+EPuxIA4cDM4vzOqOkiMPhz5XK0whEjkVzTo4+S0puvDZuwIsdiW9mxhJc7tgBNL0cYlWSYVkz4G/fslNfRPW5mYAM49f4fhtxPb5ok4Q2Lg9dPKVHO/Bgeu5woMc7RY0p1ej6D4CKFE6lymSDJpW0YHX/wqE9+cfEauh7xZcG0q9t2ta6F6fmX0agvpFyZo8aFbXeUBr7osSCJNgvavWbM/06niWrOvYX2xwWdhXmXSrbX8ZbabVohBK41 phodgson@thoughtworks.com"
+}
 
-	%v
+%v
 
-	resource "aws_spot_instance_request" "foo" {
-		ami = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
-		instance_type = "m1.small"
-		key_name = "${aws_key_pair.debugging.key_name}"
+resource "aws_spot_instance_request" "foo" {
+  ami           = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
+  instance_type = "m1.small"
+  key_name      = aws_key_pair.debugging.key_name
 
-		// base price is $0.044 hourly, so bidding above that should theoretically
-		// always fulfill
-		spot_price = "0.05"
+  // base price is $0.044 hourly, so bidding above that should theoretically
+  // always fulfill
+  spot_price = "0.05"
```

Comment:
Prefer `#` over `//` for comments here and elsewhere

Reply:
Fixed